templates/etc/nginx/sites-available/site-available.j2

```diff
@@ -19,6 +19,14 @@ server {
 }
 
 server {
+    gzip on;
```

Comment:
Not sure when you would want to disable `gzip` but maybe we should make a variable for it ?